README.md

```diff
@@ -1,72 +1,70 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Todo list App
 
-> Description the project.
 
+> This branch (todo__list) We did it by contributing with my coding partner we did in this branch the followings 👌 :
+- [x] Set up a new project with webpack.
+- [x] Create an index.html file and write your HTML markup here. Create an empty To Do List placeholder.
+- [x] Create The index.html file must be set as a template using the HTML Webpack Plugin.
+- [x] Create an index.js file and set an array of some simple to do tasks (array of objects).
+- [x]  Each task object should contain three keys
+- [x] Write a function to iterate over the tasks array and populate an HTML list item element for each task.💯
+- [x] On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.
+- [x] Create a style.css and set rules for the To Do List. CSS must be loaded by Webpack Style/CSS Loader.
 
-## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
 
-## Live Demo (if available)
+## Built With
 
-[Live Demo Link](https://livedemo.com)
+- HTML
+- CSS
+- Flexbox
+- JavaScript
+- Use the javascript objects and arrays to store the data for every project.
+- Use the webpack module bundler for setting up the project.
 
 
-## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
 
 
-To get a local copy up and running follow these simple example steps.
+## Live Demo
 
-### Prerequisites
 
-### Setup
 
-### Install
 
-### Usage
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```

Comment:
- [ ] Kindly fill out this section properly. It should indicate how to setup the project locally (ex: open in vscode and use npm install and npm start... etc) Kindly use the template provided by Microverse 👍

Reply:
- Thanks @ha-manel 🙏 for reviewing my pull request I appreciate your review 💯 
### Thanks again for reviewing my PR  and have a good day😊